ktoml-core/src/commonMain/kotlin/com/akuleshov7/ktoml/decoders/TomlAbstractDecoder.kt

```diff
@@ -32,7 +34,35 @@ public abstract class TomlAbstractDecoder : AbstractDecoder() {
     override fun decodeShort(): Short = decodePrimitiveType()
     override fun decodeInt(): Int = decodePrimitiveType()
     override fun decodeFloat(): Float = decodePrimitiveType()
-    override fun decodeChar(): Char = invalidType("Char", "String")
+    override fun decodeChar(): Char {
+        val keyValue = decodeKeyValue()
+        return when (val value = keyValue.value) {
```

Comment:
We will give users a possibility to decode Char from integer or from a literal

Reply:
char1 = 156
char2 = '5'

TOML standard does not support Char, but we will have it out of the box